### ISSUE TITLE
Syntax Error: Expected `'` false positive

### ISSUE THREAD

**Issue by EsadCetiner** [coreruleset/secrules_parsing#131]
In this PR, https://github.com/coreruleset/nextcloud-rule-exclusions-plugin/pull/161/changes#diff-b927017cb1fee736fcfe55bdef8b2ca3f4966a9896ea93d3e1e6f29487303be0R1331 the `check-syntax` action is reporting a missing `"` for this rule:
```
# Removing a word from dictionary
SecRule REQUEST_HEADERS:User-Agent "@beginsWith COOLWSD HTTP Agent" \
    "id:9508557,\
    phase:1,\
    pass,\
    t:none,\
    nolog,\
    ver:'nextcloud-rule-exclusions-plugin/1.6.0',\
    chain"
    SecRule REQUEST_FILENAME "@endsWith /apps/richdocuments/wopi/settings/upload" \
        "t:none,\
        chain"
        SecRule REQUEST_HEADERS:Content-Type "@beginsWith text/html" \
            "t:none,\
            setvar:'tx.allowed_request_content_type=%{tx.allowed_request_content_type} |text/html|',\
            setvar:tx.enforce_bodyproc_urlencoded=0"
```

The rule is syntactically correct and works both on Apache and NGINX.
